frontend/src/metabase/admin/settings/components/SettingsUpdatesForm.unit.spec.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+
+import MetabaseSettings from "../../../../metabase/lib/settings";
+import SettingsUpdatesForm from "./SettingsUpdatesForm";
+
+const elements = [
+  {
+    key: "key",
+  },
+];
+
+describe("SettingsUpdatesForm", () => {
+  it("shows custom message for Cloud installations", () => {
+    const isHostedSpy = jest.spyOn(MetabaseSettings, "isHosted");
+    isHostedSpy.mockImplementation(() => true);
+
+    render(<SettingsUpdatesForm elements={elements} />);
+    screen.getByText(/Metabase Cloud keeps your instance up-to-date/);
+    isHostedSpy.mockRestore();
```

Comment:
This spying/mocking/mockRestoring looks verbose.

For context, MetabaseSettings is exported as an uncommon instantiation of a class.
 https://github.com/metabase/metabase/blob/master/frontend/src/metabase/lib/settings.js#L293

I've tried several combinations to clean these tests up: before/afterEach with restore/clear/othersMock. 

If you happen to see a cleaner way, please let me know.